src/nunit.xamarin/Helpers/ResultSummary.cs

```diff
@@ -181,6 +184,24 @@ public int NotRunCount
         /// </summary>
         public int ExplicitCount { get; private set; }
 
+        /// <summary>
+        /// Gets the time when the tests started running
+        /// </summary>
+        public DateTime StartTime { get; private set; }
+
+        /// <summary>
+        /// Gets the time when the tests ended it's execution
```

Comment:
This comment just doesn't quite work grammatically. How about:
"Gets the time when the test suite started running"
and 
 "Gets the time when the test suite completed execution"

Reply:
A lot better!! I will add this change when I get back home.
Thanks!!